paymentsheet/src/main/java/com/stripe/android/paymentsheet/navigation/PaymentSheetScreen.kt

```diff
@@ -178,7 +179,7 @@ internal sealed interface PaymentSheetScreen {
         }
     }
 
-    data object ManageSavedPaymentMethods : PaymentSheetScreen {
+    data class ManageSavedPaymentMethods(private val interactor: ManageScreenInteractor) : PaymentSheetScreen {
```

Comment:
```suggestion
   class ManageSavedPaymentMethods(private val interactor: ManageScreenInteractor) : PaymentSheetScreen {
```

It's no longer really a data class. These interactors hold state, and don't behave like we'd expect a data class to.